project/ScalafixBuild.scala

```diff
@@ -127,35 +125,30 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     updateOptions := updateOptions.value.withCachedResolution(true),
     ThisBuild / watchTriggeredMessage := Watch.clearScreenOnTrigger,
     commands += Command.command("save-expect") { s =>
-      "unit/test:runMain scalafix.tests.util.SaveExpect" ::
+      "unit2_13Target2_13/test:runMain scalafix.tests.util.SaveExpect" ::
         s
     },
     commands += Command.command("ci-3") { s =>
-      s"""set testsInput/scalaVersion := "$scala3"""" ::
-        s"""set testsOutput/scalaVersion := "$scala3"""" ::
-        "unit/testOnly scalafix.tests.rule.RuleSuite" :: s
+      "unit2_12Target3/test" ::
```

Comment:
or `unit_213Target3`: Both should work no ?

Reply:
`unit2_13Target3` does not exist for the moment, see https://github.com/scalacenter/scalafix/pull/1528#discussion_r846677314. We could add it though, but I am not sure it brings a lot of value until/unless `scalafixScalaBinaryVersion` is updated.